utility/src/main/java/org/oppia/android/util/math/RatioExtensions.kt

```diff
@@ -21,3 +21,8 @@ fun RatioExpression.toSimplestForm(): List<Int> {
 fun RatioExpression.toAnswerString(): String {
   return ratioComponentList.joinToString(separator = ":")
 }
+
+/** Returns the greatest common divisor between two integers. */
+private fun gcd(x: Int, y: Int): Int {
```

Comment:
As we are using GCD operation in RatioExtension and FractionExtension, this might scale in the future, so can we keep this in some file called `CommonExtensions` under math package only?

Reply:
I thought about this, but my main hesitation is that it's a small method to copy & I couldn't come up with a good place to keep it (it's not really an extension, and I'd rather avoid an overly generic utility file to place it in).